exalearn-ui/src/app/components/data-table/data-table.component.ts

```diff
@@ -18,12 +18,13 @@ export class DataTableComponent implements AfterViewInit, OnInit {
 	dataSource: MatTableDataSource<PassedTest | UserBack>;
 
 	@Input() displayedColumns: string[];
-	@Input() data: object[];
-	@Input() displaySearch: boolean = true;
+	@Input() data: any;
```

Comment:
Should this really be "any"?
How come we don't know the type?

Reply:
I don't know in full, it's just that the date is then distributed to all tables, and already inside the table, the type is assigned